syz-ci/jobs.go

```diff
@@ -366,6 +366,9 @@ func (jp *JobProcessor) process(job *Job) *dashapi.JobDoneReq {
 	*mgrcfg = *mgr.managercfg
 	mgrcfg.Workdir = filepath.Join(dir, "workdir")
 	mgrcfg.KernelSrc = filepath.Join(dir, "kernel")
+	if mgr.mgrcfg.KernelSrcSuffix != "" {
+		mgrcfg.KernelSrc = filepath.Join(dir, "kernel", mgr.mgrcfg.KernelSrcSuffix)
```

Comment:
You can avoid the if-check (here and in the other place). `Join` ignores empty arguments.